content/src/content/jcr_root/apps/core/wcm/components/title/v2/title/README.md

```diff
@@ -21,6 +21,7 @@ Title component written in HTL, allowing to define a section heading.
 
 * In-place editing
 * HTML element configuration (`h1` - `h6`)
+* URL and Page Links
```

Comment:
"Linkable to content pages, external URLs or page anchors"

Reply:
Done